provider/provider.go

```diff
@@ -386,6 +386,33 @@ func (p *Provider) GetBlockHeight() (int, error) {
 	return output.Height, nil
 }
 
+// GetAllParams returns the params at the specified height
+func (p *Provider) GetAllParams(height int) (*AllParams, error) {
```

Comment:
Please make `height` an optional parameter, when it is not sent it returns the last height.